shared/src/business/useCaseHelper/serveDocumentAndGetPaperServicePdf.ts

```diff
@@ -15,12 +15,14 @@ export const serveDocumentAndGetPaperServicePdf = async ({
   applicationContext,
   caseEntities,
   docketEntryId,
+  electronicParties,
```

Comment:
does it make more sense for this just to default to 

```
electronicParties = []
```

that way something that calls it doesn't need to worry about passing in an empty array?

Reply:
If any other place calls this and does not define electronicParties it will work with the old behavior. Only when this is passed (truthy value) it will override and use new logic. By adding a default value we then need to add more if/else conditions into the method;